src/gc/unix/gcenv.unix.cpp

```diff
@@ -42,15 +42,23 @@
 #include <vm/vm_param.h>
 #endif // HAVE_XSWDEV
 
+#if HAVE_XSW_USAGE
+#include <sys/sysctl.h>
```

Comment:
`sys/sysctl.h` provides `struct xsw_usage`. I can either put that in `#ifdef __APPLE__` or just under a check for `HAVE_XSW_USAGE`.

Reply:
HAVE_XSW_USAGE sounds good to me